src/components/layout/Signup.module.css

```diff
@@ -56,7 +56,7 @@
   white-space: nowrap;
 }
 
-.input {
+.inputField {
   width: 100%;
   padding: 10px;
```

Comment:
for consistency we should stick to em

Reply:
oops, I forgot! I will do that , Thank you Laura for noticing.